wpantund.cpp fails to compile due to free() out of scope

```
wpantund.cpp: In function 'void signal_critical(int, siginfo_t*, void*)':
wpantund.cpp:296:7: error: 'stack_symbols' was not declared in this scope
  free(stack_symbols);
       ^
Makefile:752: recipe for target 'wpantund-wpantund.o' failed
```

I will submit a PR addressing this issue.

--- a/src/wpantund/wpantund.cpp
+++ b/src/wpantund/wpantund.cpp
@@ -291,9 +291,9 @@ signal_critical(int sig, siginfo_t * info, void * ucontext)
 		syslog(LOG_CRIT, "[BT] %2d: %s", i, stack_symbols[i]);
 #endif
 	}
-#endif // WPANTUND_BACKTRACE
 
 	free(stack_symbols);
+#endif // WPANTUND_BACKTRACE
 
 	_exit(EXIT_FAILURE);
 }

